### PR TITLE
TFR improvements

### PR DESCRIPTION
**Pull request by hoechenberger**
- New option to subtract evoked from epochs before doing TFR analysis
- New option to customize number of Morlet cycles
- Add ITC topoplots to report

### Before merging …

- [x] Changelog has been updated (`docs/source/changes.md`)

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -1309,6 +1310,22 @@
...
+or one per frequency, where frequencies are calculated via
+`np.arange(time_frequency_freq_min, time_frequency_freq_max)`.
+If `None`, uses
+`np.arange(time_frequency_freq_min, time_frequency_freq_max) / 3`.
+"""
+
+time_frequency_subtract_evoked: bool = False
+"""
+Whether to subtract the evoked signal (averaged across all epochs) from the
+epochs, before passing them to time-frequency analysis. Set this to `True` to
```
Comment (hoechenberger) [inline config.py:1325]:
```suggestion
epochs before passing them to time-frequency analysis. Set this to `True` to
```

**Comment by hoechenberger** [comment]
Built report:
https://22981-167957354-gh.circle-artifacts.com/0/reports/ds000247/sub-0002_ses-01_task-rest_report.html

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 typing_extensions
 psutil
-numpy
+numpy >=1.20
```
Comment (hoechenberger) [inline requirements.txt:3]: so we can use `np.typing`

File: docs/source/changes.md [hunk 3]
```diff
@@ -124,6 +124,16 @@ authors:
 - The new [`epochs_metadata_query][config.epochs_metadata_query] setting
   allows to select epochs based on metadata query strings.
   ({{ gh(495) }} by {{ authors.hoechenberger }} and {{ authors.agramfort }})
+- The new [`time_frequency_cycles`][config.time_frequency_cycles] setting
```
Comment (hoechenberger) [inline docs/source/changes.md:127]: I needed this setting so I could get our TFR step to work with the ds000247 dataset; our default value leads to errors there (signal too short)

**Review by agramfort — APPROVED** [review]
LGTM !

**Pull request merged**